resolwe_bio/tools/summarize_expression_ncRNA.py

```diff
@@ -31,8 +31,8 @@
 
 gene_id_re = re.compile(r'ID=([\w\-\.]*)')
 gene_name_re = re.compile(r'oId=([\w\-\.]*)')
-class_code_re = re.compile(r'class_code=([\w\-\.]*)')
 parent_id_re = re.compile(r'Parent=([\w\-\.]*)')
+class_code_re = re.compile(r'class_code=([\w\-\.]*)')
```

Comment:
This one is strange - it passes with new order of regex and fail with the old one:

    resolwe_bio/tools/summarize_expression_ncRNA.py:35:1: E305 expected 2 blank lines after class or function definition, found 0